shared/src/main/scala/squants/mass/AreaDensity.scala

```diff
@@ -59,6 +60,12 @@ object GramsPerSquareCentimeter extends AreaDensityUnit with UnitConverter with
   val conversionFactor = (100*100d)/1000d
 }
 
+object PoundsPerAcre extends AreaDensityUnit with UnitConverter {
+  val symbol = "lbs/acre"
```

Comment:
Minor nit - could you use the existing lbs and acre symbols? So something like:

```scala
val symbol = s"${Pounds.symbol}/${Acres.symbol}"
```

We're just trying to avoid redefining symbols.